src/api/users/src/util.js

```diff
@@ -0,0 +1,27 @@
+const LinkHeader = require('http-link-header');
+
+const { USERS_URL } = process.env;
+
+module.exports.addNextLinkHeader = function (res, users, perPage) {
+  // If there aren't any results, there's no "next" page to get
+  if (!users.length) {
+    return;
+  }
+
+  // Similarly, if the number of users is less than the perPage size,
+  // don't bother adding a next link, since there aren't going to be more.
+  if (users.length < perPage) {
+    return;
+  }
+
+  // Get the id of the last user in this page of results
+  const lastUser = users[users.length - 1];
+  const lastId = lastUser.id;
+
+  // Construct the body of the header, giving the URI to use for the next page:
+  // '<https://api.telescope.cdot.systems/v1/users?startAfter=6Xoj0UXOW3&per_page=100>; rel="next"'
+  const link = new LinkHeader();
+  link.refs.push({ uri: `${USERS_URL}?startAfter=${lastId}&per_page=${perPage}`, rel: 'next' });
```

Comment:
`"message": "\"starAfter\" is not allowed"`:

```suggestion
  link.refs.push({ uri: `${USERS_URL}?start_after=${lastId}&per_page=${perPage}`, rel: 'next' });
```

The comment needs to reference the change as well.

Reply:
Fixed in https://github.com/Seneca-CDOT/telescope/pull/2060/commits/dc9a3d07b38f2b048b9caa13d9c5459a4c300f2a